v2/CISCO-ALARM-MIB.my

```diff
@@ -46,12 +46,12 @@ ciscoAlarmMIB MODULE-IDENTITY
             USA
             Tel: +1 800 553-NETS
             E-mail: cs-<list>@cisco.com"
+    DESCRIPTION
+        "Added hwCem(68) to existing CoiAlarmObjectTypeClass object."
+    REVISION        "202105170000Z"
     DESCRIPTION
         "The MIB module describes a generic solution to model alarms
         and to store the current list of active alarms."
-    REVISION        "202008310000Z"
-    DESCRIPTION
-        "MIB changes for trap"
```

Comment:
Dont remove any revision. This is required for history.

Reply:
made required changes